test/unit/index.test.ts

```diff
@@ -141,6 +141,63 @@ describe('api routes', (): void => {
   })
 
   describe('Endpoint: /participants/{Type}/{ID}', (): void => {
+    it('should return 500 for GET /participants/MSISDN/ (missing ID - validation error)', async (): Promise<void> => {
```

Comment:
I thought it would be 400 or 422 or something like that for these kind of errors..